app.js

```diff
@@ -0,0 +1,132 @@
+/* Esta funcion simula el test de tipo de piel de la pagina. En lugar de usar checkboxes, el usuario responde preguntas (si/no). Se mantiene la logica del test original, pero adaptada a entrada/salida por promt y alert */
+
+// funcion para el test de tipo de piel
+function iniciarTest() {
+    // lista de preguntas con su tipo asociado
+    const preguntas = [
+        {
+            texto: "Tu piel se siente tirante despues de lavarla?",
+            tipo: "seca"
+        },
+        {
+            texto: "Tu piel se siente aceitosa al tacto?",
+            tipo: "grasa"
+        },
+        {
+            texto: "Tenez zonas grasas (nariz, frente) y otras secas (mejillas)?",
+            tipo: "mixta"
+        },
+        {
+            texto: "Tu piel birlla pocas horas despues de lavarla?",
+            tipo: "grasa"
+        },
+        {
+            texto: "Tu piel se ve opaca o apagada?",
+            tipo: "seca"
+        },
+        {
+            texto: "Te tenes que aplicar crema humectante varias veces al dia?",
+            tipo: "seca"
+        },
+        {
+            texto: "Tu piel se seinte comoda sin resequedad ni grasitud?",
+            tipo: "normal"
+        },
+        {
+            texto: "Tu piel se enrojece facilmente o reacciona mal a productos?",
+            tipo: "sensible"
+        },
+    ];
+
+    // objeto para contar las respuestas afirmativas por tipo
+    let conteo = {
+        seca: 0,
+        grasa: 0,
+        mixta: 0,
+        normal: 0,
+        sensible: 0
+    }
+
+    // pregunta al usuario y acumula las respuestas
+    alert("Bienvenido al Test para saber su tipo de piel, a continuacion respondera algunas preguntas");
+    for (let i = 0; i < preguntas.length; i++) {
+        let respuesta = prompt(preguntas[i].texto + " (si/no)").toLowerCase();
+        if (respuesta === "si") {
+            conteo[preguntas[i].tipo]++;
+        }
+    }
+
+    // determina el tipo de piel con las respuestas
+    let tipoFinal = "";
```

Comment:
Puede ser un método de la clase Test
```suggestion
 determinarTipoPiel() {
        let tipoFinal = "";
        let mayorCantidad = 0;

        for (let tipo in this.conteo) {
            if (this.conteo[tipo] > mayorCantidad) {
                mayorCantidad = this.conteo[tipo];
                tipoFinal = tipo;
            }
        }
        return tipoFinal;
    }
```